packages/generate-newsletter/utils.js

```diff
@@ -0,0 +1,54 @@
+const colors = require('./colors')
+
+const generateRandomNum = (min, max) => {
+  return Math.floor(Math.random() * (max - min) + min)
+}
+
+const getRandomColor = () => {
+  const randomIndex = generateRandomNum(0, colors.length - 1)
+  return colors[randomIndex - 1]
+}
+
+const getCurrentDate = () => {
+  let today = new Date()
+  let dd = today.getDate()
+  let mm = today.getMonth() + 1 //January is 0!
+  let yyyy = today.getFullYear()
+
+  if (dd < 10) {
+    dd = '0' + dd
+  }
+
+  if (mm < 10) {
+    mm = '0' + mm
+  }
+
+  return yyyy + '-' + mm + '-' + dd
+}
+
+const colorLuminance = (hex, lum) => {
```

Comment:
We are not using this anymore, Lets remove this ?

Reply:
yeah.. these should be removed. I will make those changes